src/OauthTokenFileStorage.php

```diff
@@ -139,7 +137,19 @@ public function removeToken(): void {
     // Remove the data from the static cache.
     static::$token_data = NULL;
     // Remove the token data from storage.
-    file_unmanaged_delete(static::OAUTH_TOKEN_PATH);
+    file_unmanaged_delete(static::oauthTokenPath());
+  }
+
+  /**
+   * Gets the storage location for OAuth token data.
+   */
+  public static function oauthTokenPath() {
+    static $token_path;
+
+    $token_path = $token_path
```

Comment:
It's true that using settings keeps this out of config. I thought that's what we wanted.

Reply:
No, because we would like to allow to sync this custom token location - for example when all servers on ACQ - but we do not want to expose this configuration option on the UI.

Reply:
@cnovak What do you think, should this be exportable configuration or just something that would be in `settings.php`?

Reply:
I was thinking a config object the way we discussed earlier, I don't see any issue w/making it a config value.  I was thinking it would not be in the  apigee_edge.auth.yml however since this isn't something we need to store w/secure settings, but maybe that is OK to have it there

Reply:
I changed it to `$config['apigee_edge.auth']['oauth_token_storage_location']`. There are only  two items in this config item now. The active key and the token storage location.